content/blog/lektor-32-released/contents.lr

```diff
@@ -0,0 +1,59 @@
+title: Lektor 3.2 Released
+---
+author: Andreas Runfalk
+---
+pub_date: 2020-05-11
+---
+summary: What's changed with Lektor 3.2
+---
+body:
+
+#### text-block ####
+text:
+
+It's been a long time in the making and on behalf of the Lektor Team I'm happy to introduce the 3.2 release of Lektor. As some of you may have noticed the development pace of Lektor has slowed down the two years. Those of you who have had open PRs for a long time, sorry for the delay and we'll try to do better going forward!
+
+Recently we've made a big push to bring the project back on its feet. Our focus besides merging PRs and fixing various issues have been to streamline the CI and release process. The biggest change was the [switch from Travis and Appveyor to GitHub actions](https://github.com/lektor/lektor/pull/734) which should make the release process a lot easier going forward. We are also moving all official plugins to 
+
+We have also promoted [lektor-tags](https://github.com/lektor/lektor-tags) and [lektor-atom](https://github.com/lektor/lektor-atom) to official plugins. Expect new releases of them in the near future. Before they are ready we'll deploy a [new CI workflow](https://github.com/lektor/lektor-atom/pull/28) that can publish new versions using tags. This will be something all plugin authors can benefit from going forward.
+
+!! Since Python 2 is EOL and its [final release](https://blog.python.org/2020/04/python-2718-last-release-of-python-2.html) ever happened on April 20th, 2020 this will be the last Lektor release with Python 2 support.
```

Comment:
I assume you removed the date to avoid the commas? It seems to be correct to put comma before and after the year when it comes mid sentence. I'll keep the full date in case someone reads this next year.